crates/network-scanner/src/scanner.rs

```diff
@@ -351,3 +349,39 @@ pub struct NetworkScannerParams {
     pub mdns_single_query_timeout: u64, // in milliseconds
     pub max_wait_time: u64,             // max_wait for entire scan duration in milliseconds, suggested!
 }
+
+#[derive(Debug, Clone)]
+pub enum Protocol {
+    /// Wayk Remote Desktop Protocol
+    Wayk,
+    /// Remote Desktop Protocol
+    Rdp,
+    /// Apple Remote Desktop
+    Ard,
+    /// Virtual Network Computing
+    Vnc,
+    /// Secure Shell
+    Ssh,
+    /// PowerShell over SSH transport
+    SshPwsh,
+    /// SSH File Transfer Protocol
+    Sftp,
+    /// Secure Copy Protocol
+    Scp,
+    /// Telnet
+    Telnet,
+    /// PowerShell over WinRM via HTTP transport
+    WinrmHttpPwsh,
+    /// PowerShell over WinRM via HTTPS transport
+    WinrmHttpsPwsh,
+    /// Hypertext Transfer Protocol
+    Http,
+    /// Hypertext Transfer Protocol Secure
+    Https,
+    /// LDAP Protocol
+    Ldap,
+    /// Secure LDAP Protocol
+    Ldaps,
+    /// Devolutions Gateway Tunnel (generic JMUX tunnel)
+    Tunnel,
```

Comment:
issue: The "Devolutions Gateway Tunnel" aka "JMUX" protocol is not something we can scan for. You can remove it from the enum.

Reply:
updated